snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: exa
+version: '0.7.0'
+summary: Replacement for 'ls' written in Rust
+description: |
+  It uses colours for information by default, helping you distinguish between
+  many types of files, such as whether you are the owner, or in the owning
+  group. It also has extra features not present in the original ls, such as
+  viewing the Git status for a directory, or recursing into directories with a
+  tree view. exa is written in Rust, so it’s small, fast, and portable.
+
+grade: stable
+confinement: strict
```

Comment:
Won't the `strict` confinement here mean that I can't use `exa` against anything but the files that are shipped with the snap. For use as a general system utility, I would have expected this to be a classic snap.